### PR TITLE
remove mutable args (and add six requirement)

### PR DESCRIPTION
**Pull request by tidalf**

### CODE REVIEW

File: resources/lib/qobuz/node/artist.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 logger = getLogger(__name__)
 
 
-def helper_album_list_genre(data, default=[]):
```
Comment (showi) [inline resources/lib/qobuz/node/artist.py]: must be []

File: resources/lib/qobuz/node/artist.py [hunk 2]
```diff
@@ -18,7 +18,8 @@
...
+def helper_album_list_genre(data, default=None):
```
Comment (showi) [inline resources/lib/qobuz/node/artist.py]: must be **[]**

Comment (tidalf) [reply resources/lib/qobuz/node/artist.py]: true

File: resources/lib/qobuz/registry.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ class Registry(object):
     def __init__(self, application):
         self.application = application
         try:
-            import xbmc # pylint: disable=W0612
+            import xbmc  # pylint: disable=W0612
```
Comment (showi) [inline resources/lib/qobuz/registry.py]: ws en trop

Comment (showi) [reply resources/lib/qobuz/registry.py]: je crois :)

File: resources/lib/qobuz/node/artist.py (hunk 2 above)
Comment (tidalf) [reply resources/lib/qobuz/node/artist.py]: fixed

File: resources/lib/qobuz/registry.py (hunk 3 above)
Comment (tidalf) [reply resources/lib/qobuz/registry.py]: non a priori c 2

**Pull request merged**